src/templates/talksregister.js

```diff
@@ -0,0 +1,285 @@
+              </h2>
+              <div className="row align-items-center justify-content-center">
+                <div className="col-md-6 py-3 d-flex align-items-center">
+                  <form
+                    action={"https://techlabs.org"}
+                    method="post"
+                    id="mc-embedded-subscribe-form"
+                    name="mc-embedded-subscribe-form"
+                    class="validate"
+                    target="_blank"
+                    novalidate
+                    className="w-100"
+                  >
+                    <div className="form-group">
+                      <label for="mce-FNAME">
+                        <FormattedMessage id={"talk.register.name.label"}/>
+                      </label>
+                      <input
+                        type="text"
+                        name="FNAME"
+                        className="form-control"
+                        id="mce-FNAME"
+                        placeholder={namePlaceholder}
+                        required
+                      />
+                    </div>
+                    <div class="form-group">
+                      <label for="exampleInputEmail1">
+                        <FormattedMessage id={"talk.register.email.label"}/>
+                      </label>
+                      <input
+                        type="email"
+                        className="form-control"
+                        placeholder={emailPlaceholder}
+                        id="mce-EMAIL"
+                        aria-describedby="emailHelp"
+                        name="EMAIL"
+                        required
+                      />
+                    </div>
+                    <div class="form-group">
+                      <label for="">
+                        <FormattedMessage id={"talk.register.location.label"}/>
+                      </label>
+                      <div style={{paddingTop: "5px", marginRight: "40%"}}>
+                        <select
+                          className="dropdown-select"
+                        >
+                          {data.location.nodes.map(locationoption => (
+                            <option key={locationoption.heading} value={locationoption.heading}>
+                              {locationoption.heading}
+                            </option>
+                          ))}
+                        </select>
+                      </div>
+                    </div>
+                    <div className="form-group">
+                      <label for="">
+                        <FormattedMessage id={"talk.register.participated.question"}/>
+                      </label>
+                      <div>
+                        <ul className="newsletter-checkbox mt-0">
+                          <li>
+                            <div class="row justify-content-around">
+                              <div class="col-3">
+                                <input
```

Comment:
Warum nimmst du hier denn den typ checkbox? - Macht es in dem Fall nicht viel mehr sinn einen Radio button zu nehmen? Dann hättest du auch nicht das Problem mit dem required - Also du würdest es dann einfach auf required setzen und dann muss eins von beiden ausgewählt sein

Reply:
jup hast recht, da bin ich irgendwie nicht drauf gekommen. habe ich angepasst!